nadlogar/static/formstyle.css

```diff
@@ -0,0 +1,11 @@
+
+input, optgroup, select, textarea {
+    width: -webkit-fill-available;
```

Comment:
Tole bo delovalo samo na webkit browserjih, treba bi bilo dodati še vsaj `-moz-fill-available` in splošni `fill-available`. Poleg tega je bill `fill-available` preimenovan v `stretch`, ce se prav spomnim, pa še ta je zaenkrat pod experimental. 
 https://developer.mozilla.org/en-US/docs/Web/CSS/width#browser_compatibility

Skoraj gotovo se da to narediti brez experimental featurjev?